lib/registration.pm

```diff
@@ -43,6 +44,14 @@ our %SLE15_MODULES = (
     serverapp => 'Server-Applications'
 );
 
+# The expected modules of a default installation per product. Use them if they
+# are not preselected, to crosscheck or just recreate automatic selections
+# manually
+our %SLE15_DEFAULT_MODULES = (
+    sles => 'base,script,desktop,serverapp',
```

Comment:
the script in here is actually a bug

Reply:
really? which one?

Reply:
We will discuss this once scc shows the right default modules - no need to discuss this now :)

Reply:
yes, that's what I'm thinking. Anyway I reported a bug about this. Also here we use the same list for the all-packages medium where we don't assume there are any modules pre-selected